app/models/assessment.rb

```diff
@@ -131,28 +132,7 @@ def translations
   # Returns a representative FHIR::QuestionnaireResponse for an instance of a Sara Alert Assessment.
   # https://www.hl7.org/fhir/observation.html
   def as_fhir
-    FHIR::QuestionnaireResponse.new(
-      meta: FHIR::Meta.new(lastUpdated: updated_at.strftime('%FT%T%:z')),
-      id: id,
-      subject: FHIR::Reference.new(reference: "Patient/#{patient_id}"),
-      status: 'completed',
-      item: reported_condition.symptoms.enum_for(:each_with_index).collect do |s, index|
-        case s.type
-        when 'IntegerSymptom'
-          FHIR::QuestionnaireResponse::Item.new(text: s.name,
-                                                answer: FHIR::QuestionnaireResponse::Item::Answer.new(valueInteger: s.int_value),
-                                                linkId: index.to_s)
-        when 'FloatSymptom'
-          FHIR::QuestionnaireResponse::Item.new(text: s.name,
-                                                answer: FHIR::QuestionnaireResponse::Item::Answer.new(valueDecimal: s.float_value),
-                                                linkId: index.to_s)
-        when 'BoolSymptom'
-          FHIR::QuestionnaireResponse::Item.new(text: s.name,
-                                                answer: FHIR::QuestionnaireResponse::Item::Answer.new(valueBoolean: s.bool_value),
-                                                linkId: index.to_s)
-        end
-      end
-    )
+    assessment_as_fhir(self)
```

Comment:
I'm only ever seeing this called here. Why hide this in a helper? Logically I would come here first to find how a model is serialized into FHIR, but then would need to jump to the helper to really see what's going on.

Reply:
I think this is just following a pattern I established a while ago by moving the `patient_as_fhir` method to the FhirHelper, which I did because I liked the idea of having all the FHIR-specific code in one place.

Reply:
Not sure I'm a fan of that; let's not hide this stuff in helpers unless it's a method used multiple times.

Reply:
I see your point, but I want to try to make the case- the `FhirHelper` essentially just functions as a helper for `ApiController`. The `_as_fhir` methods like this one in the models that just proxy the `FhirHelper` exist exclusively to make the semantics of the call from `ApiController` slightly nicer, i.e. `assessment.as_fhir` rather than `assessment_as_fhir(assessment)`, but we could just as easily do the latter and not involve the model classes at all. I don't feel strongly either way on keeping or removing these proxy methods.

For the sake of development, I think it's easier to have all the FHIR stuff in one place. And to me it does make sense to encapsulate all of the FHIR mapping logic in a module. I might feel different if FHIR ran through more of the application or was more central to its design, but since our usage of FHIR is isolated to the API, I like that the FHIR code is neatly separated from the models as well.

Reply:
Yeah I will say that from a dev perspective, it has been nice having everything FHIR in one place. Just makes it easy to compare the `_as_fhir` and `_to_fhir` methods, and to follow a consistent pattern in these methods across different resources. I do see the point that it's annoying for someone seeing this for the first time to follow that chain to the model and then to the helper, but on the other hand, once someone knows about `FhirHelper`, it should be really easy to find anything FHIR-related. But I can see the argument for both sides, so I won't die on this hill if you feel really strongly @holmesie.